packages/flutter/lib/src/foundation/diagnostics.dart

```diff
@@ -495,6 +638,41 @@ final TextTreeConfiguration whitespaceTextConfiguration = TextTreeConfiguration(
   isBlankLineBetweenPropertiesAndChildren: false,
 );
 
+/// Whitespace only configuration where children are consistently indented
```

Comment:
Where are the two spaces coming from? The example below shows everything at the same indentation.

Reply:
There is no indentation. Clearly I started with the comment from the whitespace style and failed to update it.

Reply:
The comment was wrong. Fixed it. All the example diagrams were carefully constructed but some comments were out of sync with the content.